.github/workflows/build-and-push.yaml

```diff
@@ -133,8 +135,10 @@ jobs:
         name: Fetch and Set Environment Variables from Secret Manager
         id: deploy-secrets
         run: |
+          # Get appropriate secret name
+          SECRET_NAME=$(gcloud secrets list --filter="labels.environment:${{ steps.deploy-info.outputs.environment_name }} AND labels.deployment:${{ steps.deploy-info.outputs.deployment }}" --project=${{ steps.deploy-info.outputs.project }} --format="value(name)")
```

Comment:
I _think_ you should be okay not specifying the environment/project for the same reason below